Cargo.toml

```diff
@@ -71,6 +71,7 @@ default = ["kvm", "io_uring"]
 dbus_api = ["zbus", "vmm/dbus_api"]
 dhat-heap = ["dhat"] # For heap profiling
 guest_debug = ["vmm/guest_debug"]
+igvm = ["vmm/igvm"]
```

Comment:
Please see the comments above the features list and make changes to `vmm::feature_list()` accordingly.